### PR TITLE
Updated MySQL version in Readme

### PR DESCRIPTION
**Pull request by as666499**
**Description of the change**

**Benefits**

**Drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: charts/portal/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This Chart deploys the Layer7 API Developer Portal on a Kubernetes Cluster using
 Solutions & Patches](https://techdocs.broadcom.com/us/product-content/recommended-reading/technical-document-index/ca-api-developer-portal-solutions-and-patches.html)
 
 ### Production
-- A dedicated MySQL 5.6/7 server [TechDocs](https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-0/install-configure-and-upgrade/install-portal-on-docker-swarm/configure-an-external-database.html#concept.dita_18bc57ed503d5d7b08bde9b6e90147aef9a864c4_ProvideMySQLSettings)
+- A dedicated MySQL 8.22 server [TechDocs](https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-0/install-configure-and-upgrade/install-portal-on-docker-swarm/configure-an-external-database.html#concept.dita_18bc57ed503d5d7b08bde9b6e90147aef9a864c4_ProvideMySQLSettings)
```
Comment (APIM-KiranVaddadi) [inline charts/portal/README.md]: We support both 5.6/7 and 8.22

File: charts/portal/values-production.yaml [hunk 2]
```diff
@@ -743,10 +743,17 @@ ingress-nginx:
 # in any production environment.
 # MySQL Stable Chart values - https://github.com/bitnami/charts/tree/master/bitnami/mysql
 mysql:
-  imageTag: "8.0.22-debian-10-r75"
+  image:
+    tag: "8.0.22-debian-10-r75"
   auth:
     username: portal
     existingSecret: database-secret
+  # podSecurityContext:
```
Comment (APIM-KiranVaddadi) [inline charts/portal/values-production.yaml]: podSecurity is under primary
https://github.com/bitnami/charts/blob/master/bitnami/mysql/values.yaml

File: charts/portal/values.yaml [hunk 3]
```diff
@@ -590,10 +590,17 @@ ingress-nginx:
 # in any production environment.
 # MySQL Stable Chart values - https://github.com/bitnami/charts/tree/master/bitnami/mysql
 mysql:
-  imageTag: "8.0.22-debian-10-r75"
+  image:
+    tag: "8.0.22-debian-10-r75"
   auth:
     username: portal
     existingSecret: database-secret
+  # podSecurityContext:
```
Comment (APIM-KiranVaddadi) [inline charts/portal/values.yaml]: Same as above.

**Review by APIM-KiranVaddadi — CHANGES_REQUESTED** [review]

File: charts/portal/README.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ This Chart deploys the Layer7 API Developer Portal on a Kubernetes Cluster using
 Solutions & Patches](https://techdocs.broadcom.com/us/product-content/recommended-reading/technical-document-index/ca-api-developer-portal-solutions-and-patches.html)
 
 ### Production
-- A dedicated MySQL 5.6/7 server [TechDocs](https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-0/install-configure-and-upgrade/install-portal-on-docker-swarm/configure-an-external-database.html#concept.dita_18bc57ed503d5d7b08bde9b6e90147aef9a864c4_ProvideMySQLSettings)
```
Comment (APIM-KiranVaddadi) [inline charts/portal/README.md:17]: Tech docs need to point to 5.0CR1

**Review by APIM-KiranVaddadi — CHANGES_REQUESTED** [review]

**Review by APIM-KiranVaddadi — COMMENTED** [review]
LGM

**Review by APIM-KiranVaddadi — APPROVED** [review]

**Review by Gazza7205 — APPROVED** [review]

**Pull request merged**